runeq/v2sdk/stream.py

```diff
@@ -6,7 +6,7 @@
 from . import client
 
 
-def get_stream(stream_id: str, **params):
+def get_stream(stream_id: str, limit: int = 100, format: str = "csv"):
```

Comment:
`format` can be `Literal["csv", "json"]` (from package `typing`) for the type annotations to enforce those two values specifically when doing static analysis.

Reply:
Looks like this isn't supported in python 3.6 and 3.7, which the SDK wants to support.

Reply:
Opted to raise a value error as an alternative implementation:
```
format_types = ['csv', 'json']
if format not in format_types:
     raise ValueError("Invalid format type. Expected one of: %s" % format_types)
```